app/src/main/java/com/myperishableplanner/v21001/MainActivity.kt

```diff
@@ -177,7 +177,7 @@ class MainActivity : ComponentActivity() {
 
 
     @Composable
-    fun ExpirationFacts(items: List<Item> = ArrayList<Item>(), Categories : List<Category> = ArrayList<Category>()) {
+    fun ExpirationFacts(items: List<Item> = ArrayList<Item>(), Categories: List<Category> = ArrayList<Category>()) {
```

Comment:
Proper coding convention states not to put a space before the colon when used for a declaration and its type.